controllers/MatchUsers.py

```diff
@@ -87,4 +89,68 @@ def MatchUsers():
     # Get matches result
     res = matchingAlgorithm(curr_list, result)
 
-    return jsonify(res), 200
\ No newline at end of file
+    trueRes = []
+    for user in res:
+        # For each user that we matches the in terms of distance
+        # get their MatchUsersSchema
+        currUser = schema.load(user)
+        # Extract the user id from the schema
+        currId = currUser["UserId"]
+        # cross reference userid with the db and append the users 
+        # object to the trueRes list
+        trueRes.append(Users.query.filter_by(id=currId).first())
+
+    # Use the UsersSchema to dump and jsonify the user details
+    return jsonify(UserSchema(many=True).dump(trueRes)), 200
+
+@app_match_users.route("/create-n-matching-journeys",methods=['POST'])
+def CreateNMatchingJourneys():
```

Comment:
Could move it to a separate file. Created it because its cumbersome and repetitive to go to postman and individually create the journeys when you're not testing for a specific case but still require n matching journeys.